.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,4 +21,11 @@ admin: messed with admin stuff
 server: something server ops should know
 /:cl:
 
-[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
+<!-- both :cl:'s are required for it to work! You can put your name beside the first :cl: if you want to overwrite your github username -->
```

Comment:
Capitalize both and github

Reply:
```suggestion
<!-- Both :cl:'s are required for it to work! You can put your name beside the first :cl: if you want to overwrite your Github username -->
```

Reply:
i forgot to do it but there should be a period at the end of the sentence too please do it sweetie 😌